src/streams/TopicItem.js

```diff
@@ -2,14 +2,15 @@
 import React, { PureComponent } from 'react';
 import { StyleSheet, View } from 'react-native';
 
-import { BRAND_COLOR } from '../styles';
+import { BRAND_COLOR, SPACING } from '../styles';
 import { RawLabel, Touchable, UnreadCount } from '../common';
 
 const styles = StyleSheet.create({
   row: {
     flexDirection: 'row',
     alignItems: 'center',
-    padding: 8,
+    height: 40,
+    paddingLeft: SPACING,
```

Comment:
I guess it will affect in other list too. As `TopicItem` is used in other places too.
If all items have `paddingLeft: SPACING,` then it's better to it on the list itself. It reduces lots of computation at the time of rendering.

Reply:
Checked other places where `TopicItem` is used, there is no negative imapct. Removing padding from `TopicItem` however breaks these screens as well as prevents the border from reaching the full width of the screen.

Reply:
We don't need lines as separators.